tests/test_all.py

```diff
@@ -344,7 +344,8 @@ def test_signup_submit_same_email(self):
         )
         self.assertIn('Email exists on other user', response.content.decode())
 
-    def test_invite_submit_same_email(self):
+    def test_invite_user_duplicate_email(self):
+        """" Test for Inivtation email matching current User email """
```

Comment:
Looking through the preexisting files in this repo, I see there is not a consistent inclusion of docstrings and syntax varies, but I recommend triple quotes (note 4 opening quotes here), no space following or leading into the triple quotes of the docstring, and use a period (also note spelling typo).
```suggestion
        """Test for Invitation email matching current User email."""
```